src/core/server.cc

```diff
@@ -388,6 +388,32 @@ InferenceServer::HandleStatus(
   }
 }
 
+void
+InferenceServer::HandleControl(
```

Comment:
This needs to be updated to match the new style with TRTSERVER API. There should be both a ModelLoad and ModelUnload function instead of combining them.

For example should return Status, remove RequestStatus arg, ...